src/pages/app.tsx

```diff
@@ -238,19 +238,18 @@ const PlatformsInstallSnippet: FunctionComponent = () => (
 
 const DownloadSection: FunctionComponent = () => (
     <>
-        <DownloadLink
-            href="https://sourcegraph.com/search"
+        <Link
```

Comment:
This needs to remain `<DownloadLink>` with `downloadName` field because it's important for tracking downloads

Reply:
Thanks, it's resolved now.